packages/terra-form-select/tests/jest/__snapshots__/Frame.test.jsx.snap

```diff
@@ -212,11 +207,11 @@ exports[`Frame correctly applies the theme context className for search variant
       class="content"
     >
       <input
-        aria-describedby="terra-select-screen-reader-description-14"
+        aria-describedby="terra-select-screen-reader-description-13"
         aria-disabled="false"
         aria-expanded="false"
         aria-invalid="false"
-        aria-label="Terra.form.select.ariaLabel"
+        aria-label="Terra.form.select.ariaLabel undefined"
```

Comment:
undefined should not be part of Aria-label

Reply:
Modified.